README.md

```diff
@@ -1,26 +1,11 @@
-# Ember Data ActiveModel Adapter [![Build Status](https://travis-ci.org/adopted-ember-addons/active-model-adapter.svg?branch=master)](https://travis-ci.org/adopted-ember-addons/active-model-adapter)
```

Comment:
restore adopted-ember-addon references in the links

Reply:
Done!

Reply:
Done!

Reply:
the new github badge needs `<OWNER>` and `<REPOSITORY>` replaced in the second link